### PR TITLE
Document --log-level

### LINKED ISSUE #2245: --log-level is not in weave usage
It is mentioned at https://www.weave.works/documentation/net-1-5-0-troubleshooting/ and https://www.weave.works/documentation/net-1-5-0-plugin/net-1-5-0-plug-in-command-line/

I think it should be in the usage summary, and also specifically documented for the proxy.

### PR DESCRIPTION
**Pull request by bboreham**
Also a couple of other options on the proxy that were not clearly documented.

CC @abuehrle 

Fixes #2245

### CODE REVIEW

File: site/weave-docker-api/using-proxy.md [hunk 1]
```diff
@@ -53,6 +53,13 @@ By default, multicast traffic is routed over the Weave network.
 To turn this off, for example, because you want to configure your own multicast
 route, add the `--no-multicast-route` flag to `weave launch-proxy`.
 
+###Other Weave Proxy options
+
+ * `--without-dns` -- stop telling containers to use [WeaveDNS](/site/weavedns.md)
+ * `--log-level=debug|info|warning|error` -- controls how much
+   information to emit for debugging.
+ * `--no-restart` -- remove the default policy of `--restart=always`, if
+   you want to control start-up of the proxy yourself
```
Comment (abuehrle) [inline site/weave-docker-api/using-proxy.md]: missing punctuation at end of sentence.

**Comment by awh** [comment]
@abuehrle when the build passes you should be able to view the doc changes at https://dev-weavewww.pantheonsite.io/docs/net/2245-document-log-level/introducing-weave/

File: site/weave-docker-api/using-proxy.md [hunk 2]
```diff
@@ -53,6 +53,13 @@ By default, multicast traffic is routed over the Weave network.
 To turn this off, for example, because you want to configure your own multicast
 route, add the `--no-multicast-route` flag to `weave launch-proxy`.
 
+###Other Weave Proxy options
+
+ * `--without-dns` -- stop telling containers to use [WeaveDNS](/site/weavedns.md)
+ * `--log-level=debug|info|warning|error` -- controls how much
+   information to emit for debugging.
+ * `--no-restart` -- remove the default policy of `--restart=always`, if
+   you want to control start-up of the proxy yourself.
```
Comment (rade) [inline site/weave-docker-api/using-proxy.md]: nitpick... but shouldn't the first bullet point have a full stop too? It's not a sentence though, but neither is the 2nd. And none of the "sentences" start with a capital letter; should they? @abuehrle

Comment (abuehrle) [inline site/weave-docker-api/using-proxy.md]: The full stops should be removed, since these are not full sentences.

**Pull request merged**